doc/changes/changes_1.0.3.md

```diff
@@ -0,0 +1,17 @@
+# Parquet for Java 1.0.3, released 2021-08-05
+
+Code name: Excluded vulnerable dependency
+
+## Summary
+
+This releases excludes a vulnerable transitive dependency and adds updated version of it.
```

Comment:
maybe you could mention the CVE numbers here

Reply:
Added